Documents/Help/Changelog.md

```diff
@@ -11,6 +11,7 @@
 =======
 
 ## 2023-11-14 - Build 2311 - November 2023
+* Implemented [#1117](https://github.com/Krypton-Suite/Standard-Toolkit/issues/1117),  Is it possible to have the KForm back colour as the KPanel colour
```

Comment:
Might be better to put this under the 'November 2024' heading?

Reply:
Why? It's a nice to have in the current proposed release?

Reply:
Due to the failure to have the panel implemented by default, which was requested a long time ago.

Reply:
> Why? It's a nice to have in the current proposed release?

Is it possible to tell me which files have been altered to introduce this change, so that I can copy them over to the `pre-release` branch, as there are some v90 elements in the `alpha` branch already?

Reply:
@Smurf-IV Is it just `VisualForm`, `KryptonForm` + Readme which forms of #1117?

Reply:
Yes, I think that is the minimum

Reply:
Will do that today